File: AspNetMvcUnleashed/0672329980_source/Chapter06/CS/MvcApplication1/MvcApplication1/Helpers/SubmitButtonHelper.cs
using System;
using System.Web.Mvc;

namespace Helpers
{
    public static class SubmitButtonHelper
    {
        /// <summary>
        /// Renders an HTML form submit button
        /// </summary>
        public static string SubmitButton(this HtmlHelper helper, string buttonText)
        {
            return String.Format("<input type=\"submit\" value=\"{0}\" />", buttonText);
        }

    }
}

File: AspNetMvcUnleashed/0672329980_source/Chapter06/VB/MvcApplication1/MvcApplication1/Helpers/SubmitButtonHelper.vb
Public Module SubmitButtonHelper

	''' <summary>
	''' Renders an HTML form submit button
	''' </summary>
	<System.Runtime.CompilerServices.Extension> _
	Function SubmitButton(ByVal helper As HtmlHelper, ByVal buttonText As String) As String
		Return String.Format("<input type=""submit"" value=""{0}"" />", buttonText)
	End Function

End Module
